### DISCUSSION TITLE
Importing CSV/TSV appears successful but table is empty

### DISCUSSION THREAD

**Discussion by tobier** [supabase/supabase#21048] [Questions]
I'm importing a TSV file with 1006 rows into a table I created using the dashboard, but it seems like it silently fails. The preview looks good and I get a toast saying "Successfully imported 1006 rows" but when I query the table it is empty?

I have tried with out without RLS but no difference.

![image](https://github.com/supabase/supabase/assets/162911/01b3e72f-72b2-4981-adef-ac16814745f2)
![image](https://github.com/supabase/supabase/assets/162911/609d0764-d5cc-42d5-b1a8-13b5d6eb5377)

**Comment by ghidalg0** [comment]
Same with CSV on our side

**Reply by tobier** [reply]
@ghidalg0 good to know that it's not only me!

**Reply by GaryAustin1** [reply]
If you all have new instances try clearing the browser cache and resetting local storage.
There is/was an issue creating a table on a new instance but it errors for other users (not involving import).
You could also check the postgres logs for error.

EDIT: Note I just re-read that your table is created but is empty, so probably not related.

 https://github.com/orgs/supabase/discussions/20984 
 A fix has supposedly been rolled out.

**Reply by tobier** [reply]
@GaryAustin1 cleaned cache, did the procedure again, successful toast but no data is in the table according to the dashboard.

**Reply by ghidalg0** [reply]
@GaryAustin1 I confirm that inserting rows to a table using the csv import is not working, although I get a Toast that it worked

**Reply by rosdec** [reply]
Hi, I'm using Brave and it's pretty strong with cookies and stuff .. I've just temporarily disabled the AdBlock, and the import worked! So it might be applicable to you as well.
